ex1/ex1.c

```diff
@@ -5,10 +5,25 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include <sys/types.h>
+#include <sys/wait.h>
 
 int main(int argc, char *argv[])
 {
-    // Your code here
+    pid_t FPID = fork();
+    pid_t PID = getpid();
+    printf("Starting... %d", PID);
+    pid_t PPID = getppid();
 
```

Comment:
You should initialize a variable here and then change it in the child and parent processes.

Reply:
I have updated to reflect this request.